eth/integrity/e3_history_no_system_txs.go

```diff
@@ -0,0 +1 @@
+package integrity
```

Comment:
is this package and file needed or can remove?

Reply:
I have content for this file in `e35` branch. But can't copy it to devel. Let's leave it.